### ISSUE TITLE
Yellow Ribbon MVP: Full Accessibility & 508 Office Audit

### ISSUE THREAD

**Issue by stephorkandatx** [department-of-veterans-affairs/va.gov-team#9723]
## Steps to complete the Full Accessibility & 508 Audit: 
- [X] Product Manager: create issue and fill in feature name in the title and other bolded information appropriately
- [X] Practice area reviewers create sub-issues attached to this one with feedback for your team on *feature name*
- [ ] **Requesting team** completes attached feedback tickets based on definition of done included in each practice area feedback ticket (may differ for each practice area, please refer to attached tickets for further instruction)

## Artifacts - _please provide the following artifacts when you request a full review_
* Access information for the tool or feature in production
    * https://www.va.gov/education/yellow-ribbon-participating-schools/
    * https://www.va.gov/education/yellow-ribbon-participating-schools/
    * https://www.va.gov/education/about-gi-bill-benefits/post-9-11/yellow-ribbon-program/#learn-more-about-the-yellow-ri

* Certify that the following required accessibility tests are complete by checking each of these boxes (your accessibility specialist is available to help). Recommended items are optional, but highly encouraged.

  ### Required

    - [x] Conduct axe browser scan, and document or address issues found 
    - [x] Ensure end-to-end tests - including the axe check, pass 
    - [x] Check color pairing for proper contrast 
    - [x] Check for colorblindness issues 
    - [x] Zoom to 200% and confirm readability, no info cut off, etc 
    - [x] Repeat for 300% 
    - [x] Repeat for 400% 
    - [x] Navigate all pages and core functionality with keyboard only 
    - [x] Test all pages and core functionality happy path with MacOs + VoiceOver and/or NVDA+Firefox 

  ### Recommended

  - [x] Read the [accessibility post-launch processes](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/post-launch-audit-processes.md) 
  - [ ] Add keyboard end-to-end tests in CI/CD test runs 
  - [x] Conduct axe-coconut browser scan, document or address issues found 
  - [ ] Test with [WAVE](https://wave.webaim.org/) tool 

  ### Known issues (if applicable)
  If the team uncovers any issues during their accessibility testing, an issue should be opened and listed below, along with any other outstanding individual issues from previous reviews.

* [#8791 508-defect-3 [COGNITION, SEMANTIC MARKUP]: individual search items SHOULD read semantically](https://github.com/department-of-veterans-affairs/va.gov-team/issues/8791)

* [#9374 508-defect-3 [FOCUS MANAGEMENT]: Focus order SHOULD flow in the logical reading order](https://github.com/department-of-veterans-affairs/va.gov-team/issues/9374)

## Additional Notes
As part of this review, the VSA accessibility specialist will send your product to VA's 508 office for approval (as applicable)

**Comment by stephorkandatx** [comment]
@jenstrickland FYSA ticket has been created

**Comment by jenstrickland** [comment]
Thanks, @stephorkandatx ! 😃  Will your team please run through the checklist, check off completed items, document any known issues in issue tickets, and provide a linked list of known issues?

**Comment by stephorkandatx** [comment]
Whoever picks this up - please sync with @jenstrickland before beginning to confirm the scope of what's needed + output for the checklist. I spoke with her and she thinks we can make this easier to get completed

**Comment by MickinSahni** [comment]
Hi @jenstrickland, completed the checklist above and adding you as an assignee

**Comment by jenstrickland** [comment]
@1Copenut I finished my review of this work, and am ready to send to the VA 508 Office. Do you want to review before I do so?

**Comment by 1Copenut** [comment]
@jenstrickland I concur with your findings 💯 

I left one comment about the search count for you to review ( https://github.com/department-of-veterans-affairs/va.gov-team/issues/12060#issuecomment-675145572 ). Am happy to discuss more as you see fit.

**Comment by jenstrickland** [comment]
Sent to VA 508 Office for audit on Wednesday, August 19, 2020. Will update when I receive their report.

**Comment by MarciMcGuireGCIO** [comment]
Moved all the issues associated with this epic to a consolidated 508 Yellow Ribbon epic to clean up our backlog.
